tests/robots/test_robot.py

```diff
@@ -3,6 +3,7 @@
 import re
 
 import pytest
+from compas.data import json_dumps, json_loads
```

Comment:
not so important cause unittest, but generally one import per line

Reply:
Fixed